src/files.js

```diff
@@ -143,18 +143,19 @@ export function trashById (cozy, id) {
   return cozyFetchJSON(cozy, 'DELETE', `/files/${encodeURIComponent(id)}`)
 }
 
-export function statById (cozy, id, offline = true) {
+export function statById (cozy, id, offline = true, options = null) {
```

Comment:
What about `options = {}` ? To explicit the expected format and avoid `undefined variable options` when someone someday will do `if (options.foo)`